references/identity-facade/pipeline.sh

```diff
@@ -0,0 +1,356 @@
+#!/bin/sh
```

Comment:
if POSIX compliance makes the pipeline too unreadable - change this shebang to #!/bin/bash